acceptance-tests/build.gradle

```diff
@@ -48,13 +53,28 @@ dependencies {
 
 test.enabled = false
 
-task acceptanceTest(type: Test) {
+task downloadHashicorpVault(type: Download) {
+  src {
+    // see gradle.properties for Hashicorp Vault URL
+    return project.property("hashicorpVaultUrl.${osdetector.os}")
+  }
+  dest new File(buildDir, "hashicorp.zip")
+  onlyIfModified true
+}
+
+task downloadAndExtractHashicorpVault(dependsOn: downloadHashicorpVault, type: Copy) {
+  from zipTree(downloadHashicorpVault.dest)
+  into buildDir
+}
+
+task acceptanceTest(dependsOn: [rootProject.installDist, downloadAndExtractHashicorpVault], type: Test) {
```

Comment:
do we clean up Hashicorp on build-clean? I.e. it'd be nice not to leave it lying around on local box